### DISCUSSION TITLE
Mutable Scenarios

### DISCUSSION THREAD

**Discussion by VoVAllen** [v6d-io/v6d#277] [Q&A] [answered]
Hi,

Can I use v6d to host a shared embedding table across different instances? And each entry can be updated separately?

Thanks.

**Comment by sighingnow** [comment, marked as answer]
Sorry for the late response, I missed the message!

Yes, the shared embedding table can be shared across different instances, but vineyard's object is designed to be as immmutable, thus it cannot be updated once been sealed.

**Comment by sighingnow** [comment]
Hi @VoVAllen,

I would like to know if vineyard still in your consideration and whether the above answer is helpful for you?

Thanks!
